LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2011 Daniel Fischer, 2016-2017 Andrew Lelechenko, Carter Schonwald
+Copyright (c) 2011 Daniel Fischer, 2016-2017 Andrew Lelechenko, Carter Schonwald, Google Inc.
```

Comment:
Please bring it into sync with a copyright in https://github.com/cfredric/arithmoi/blob/5698fd8a827c3d123ce6452683310ab4936bcaaf/Math/NumberTheory/GaussianIntegers.hs#L3